frontend/src/Components/BusinessDetails/BusinessDetails.jsx

```diff
@@ -89,8 +119,36 @@ const BusinessDetails = () => {
             </Modal>
           )}
           <div className="business-main-content">
-            <div className="business-section business-overview">
-              <BusinessOverview overview={business.overview} />
+            <div className="business-section business-map">
+              <APIProvider apiKey={apiKey}>
+                <Map
+                  defaultZoom={20}
+                  defaultCenter={parseLocation(business.location)}
```

Comment:
nit: we call `parseLocation(business.location)` in 3 places.  As a small performance optimization, you could instead do this once and store the result in a variable.

Reply:
Noted

Reply:
Thank you!